guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -105,7 +105,8 @@
                         "pt-br-qwerty",
                         "sv-se-qwerty",
                         "da-dk-qwerty",
-                        "tr-tr-qwerty"
+                        "tr-tr-qwerty",
+                        "nb-no-qwerty"
```

Comment:
I know the overall list lacks alphabetical order, but it'd be nice if the new entry were in the best possible place (between `ja-jp-qwerty` and above `pt-br-qwerty`).